pkg/constants/main.go

```diff
@@ -31,13 +32,19 @@ const (
 	DefaultMergedPRsRedLevel            = 7.0
 	DefaultMergedPRsNoRetestYellowLevel = 0.75
 	DefaultMergedPRsNoRetestRedLevel    = 0.5
+	DefaultSIGRetestYellowLevel         = 10
```

Comment:
Maybe use 1 here? Since if there's a retest failure we don't want to show this as green, no?
```suggestion
	DefaultSIGRetestYellowLevel         = 1
```